safety_down_under/content/_index.md

```diff
@@ -2,6 +2,10 @@
 title: 'Home'
 date: 2018-02-12T15:37:57+07:00
 heroHeading: 'Safety Down Under, Inc.'
-heroSubHeading: 'CPSI: Certified Playground Safety Inspectors. We provide playground inspections, maintenance, repair, installations, and engineered wood fiber safety surfacing.'
+heroSubHeading: 'Playground Safety Specialists. Certified WBENC
+Women's Business Enterprise. Certified DBE Disadvantaged Business Enterprise.
+Providing playground safety surfacing, playground inspections, 
+playground maintenance, playground design and installation, gymnasium
+inspections and maintenance, fencing, and bulk landscape supplies.'
```

Comment:
This didn't *need* to be split into multiple lines but I think it makes sense to do so. That said, there's a few things here that need to change. to do that correctly, the `heroSubHeading` needs to be prepended with a `>`:

so:

```suggestion
heroSubHeading: >
  Playground Safety Specialists. Certified WBENC
  Women's Business Enterprise. Certified DBE Disadvantaged Business Enterprise.
  Providing playground safety surfacing, playground inspections, 
  playground maintenance, playground design and installation, gymnasium
  inspections and maintenance, fencing, and bulk landscape supplies.
```